lib/presentation/core/help_center/pages/help_center_page.dart

```diff
@@ -17,7 +17,12 @@ import 'package:shared_themes/text_themes.dart';
 class HelpCenterPage extends StatelessWidget {
   final ReduxAction<CoreState>? logoutAction;
 
-  const HelpCenterPage({Key? key, this.logoutAction}) : super(key: key);
+  final String? title;
+
+  const HelpCenterPage({
+    this.logoutAction,
+    this.title,
```

Comment:
can we have the app context passed in here instead of the title?

we want to cater for a scenario in the future where the help center phone and WhatsApp number could also change